apps/src/javalab/DefaultVisualization.jsx

```diff
@@ -1,14 +1,53 @@
 import React from 'react';
-import PaneHeader, {PaneSection} from '@cdo/apps/templates/PaneHeader';
+import PaneHeader, {
+  PaneSection,
+  PaneButton
+} from '@cdo/apps/templates/PaneHeader';
+import CollapserIcon from '@cdo/apps/templates/CollapserIcon';
 import ProtectedVisualizationDiv from '@cdo/apps/templates/ProtectedVisualizationDiv';
 
 export default class DefaultVisualization extends React.Component {
+  state = {
+    isCollapsed: false,
+    isFullscreen: false
+  };
+
+  fullscreenIcon = () => {
+    return this.state.isFullscreen ? 'fa fa-compress' : 'fa fa-arrows-alt';
+  };
+
   render() {
     return (
       <PaneHeader hasFocus>
+        <PaneButton
+          headerHasFocus
+          icon={<CollapserIcon isCollapsed={this.state.isCollapsed} />}
+          onClick={() => {}}
+          label=""
+          isRtl={false}
+          style={styles.transparent}
+          leftJustified
+        />
+        <PaneButton
+          headerHasFocus
+          iconClass={this.fullscreenIcon()}
+          onClick={() => {}}
```

Comment:
My understanding is preview fullscreen won't be enabled until post-pilot. Do we still want this button here if it won't do anything for a long time? @jmkulwik is my understanding correct?

Reply:
That is true. Let me check in with @moneppo

Reply:
we're going to hide the fullscreen icon for now ([slack thread](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1622142834164000)). removing this element causes the "Preview" text to not be centered in the header, so i added an empty placeholder icon to keep the text centered. when we're ready to implement fullscreen, we can add `iconClass` back and remove `icon` here